taf/keystore.py

```diff
@@ -94,11 +93,6 @@ def read_private_key_from_keystore(
     if not key_path.is_file():
         raise KeystoreError(f"{str(key_path)} does not exist")
 
-    if password is not None and roles_key_info is not None:
-        passwords = roles_key_info.get("passwords")
-        if passwords is not None and len(passwords):
-            password = passwords[key_num]
-
```

Comment:
Why is this not needed?

Reply:
we don't set passwords, but even if we did, seems like a bad idea to store them in files like that. So I thought it made sense to just remove that

Reply:
Makes sense to me